src/packages/client/src/__tests__/atomicOperationsUpdate.test.ts

```diff
@@ -0,0 +1,85 @@
+import chalk from 'chalk'
+import { blog } from '../fixtures/blog'
+import { DMMFClass, makeDocument, transformDocument } from '../runtime'
+import { getDMMF } from '../generation/getDMMF'
+chalk.level = 0
+
+let dmmf
+describe('minimal where transformation', () => {
+  beforeAll(async () => {
+    dmmf = new DMMFClass(await getDMMF({ datamodel: blog }))
+  })
+
+  test('OR posts some id in', () => {
+    function getTransformedDocument(select) {
+      const document = makeDocument({
+        dmmf,
+        select,
+        rootTypeName: 'mutation',
+        rootField: 'updateOneUser',
+      })
+      return String(transformDocument(document))
+    }
+
+    const transformedDocument = getTransformedDocument({
+      data: {
+        countInt1: {
+          set: null,
+        },
+        countInt2: {
+          set: 123,
+        },
+        countInt3: {
+          increment: 1,
+        },
+        countInt4: {
+          decrement: 1,
+        },
+        countInt5: {
+          multiply: 2,
+        },
+        countInt6: {
+          divide: 4,
+        },
```

Comment:
It might be good to also add one more variant to also test the vanilla case without a nested object. E.g. `countInt7: 99`. To make sure this gets wrapped in an object with `set: `

Reply:
Btw this approach looks neat in order to test the DMMF transformations. 👍

Reply:
Ok I will do that! Thanks for reviewing 💯

Reply:
It looks ok 👍 

```
 "mutation {
        updateOneUser(
          data: {
            countFloat: {
              set: 3.1415926
            }
            countInt1: {
              set: 3
            }
          }
        ) {
          id
          email
          name
          json
          countFloat
          countInt1
          countInt2
          countInt3
          countInt4
          countInt5
          countInt6
        }
      }"
```